src/less/carousel/stories/carousel.stories.js

```diff
@@ -29,6 +29,35 @@ export const continuous = () => `
 </div>
 `;
 
+export const imageTreatment = () => `
+<div class="carousel">
+    <div class="carousel__container carousel__container--image-treatment">
+        <button class="carousel__control carousel__control--prev" aria-label="Previous Slide - Top Products">
+            <svg aria-hidden="true" class="icon icon--carousel-prev" focusable="false">
+                <use xlink:href="#icon-carousel-prev"></use>
+            </svg>
+        </button>
+        <div class="carousel__viewport carousel__viewport--mask">
+            <ul class="carousel__list carousel__list--default-demo">
+                <li><img src="https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Ftse3.mm.bing.net%2Fth%3Fid%3DOIP.HxJ4seuyUbgl_6ZTuZzbywHaE8%26pid%3DApi&f=1" /></li>
+                <li><img src="https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Ftse1.mm.bing.net%2Fth%3Fid%3DOIP.ryczKwfKPHOEUpqsspzn8AAAAA%26pid%3DApi&f=1" /></li>
+                <li><img src="https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Ftse2.mm.bing.net%2Fth%3Fid%3DOIP.uaG09RTqJi-901pnY2SB5AHaE6%26pid%3DApi&f=1" /></li>
+                <li><img src="https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Ftse1.mm.bing.net%2Fth%3Fid%3DOIP.SN0Lv4ITuUd4rRenyzgeZgHaHa%26pid%3DApi&f=1"/></li>
+                <li><img src="https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Ftse3.mm.bing.net%2Fth%3Fid%3DOIP.4IFrCamfbODucINDz8VGnQAAAA%26pid%3DApi&f=1" /></li>
+                <li><img src="https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Ftse4.mm.bing.net%2Fth%3Fid%3DOIP.1R4ymUtVrFa0fESEwSBLlwHaDO%26pid%3DApi&f=1"/></li>
+                <li>Card 7</li>
+                <li>Card 8</li>
```

Comment:
I would figure out a way to access these images differently. Maybe put them in a CDN link or something. They might expire and might be not found

Reply:
that's a good point. Do we have a CDN I could use?